JPS_VIRTUALSENSOR/JurongIslandInputAgent/src/main/java/com/cmclinnovations/jurongisland/EndpointConfig.java

```diff
@@ -0,0 +1,49 @@
+package com.cmclinnovations.jurongisland;
+
+import com.cmclinnovations.stack.clients.blazegraph.BlazegraphEndpointConfig;
+import com.cmclinnovations.stack.clients.docker.ContainerClient;
+import com.cmclinnovations.stack.clients.postgis.PostGISEndpointConfig;
+
+public class EndpointConfig {
+    private String dburl;
+	private String dbuser;
+	private String dbpassword;
+    
+	private String kgurl;
+	private String kguser;
+	private String kgpassword;
+    
+    public EndpointConfig() {
+        ContainerClient containerClient = new ContainerClient();
+        PostGISEndpointConfig postGISEndpointConfig = containerClient.readEndpointConfig("postgis",
+                    PostGISEndpointConfig.class);
+        this.dburl = postGISEndpointConfig.getJdbcURL(EnvConfig.DATABASE);
+        this.dbuser = postGISEndpointConfig.getUsername();
+        this.dbpassword = postGISEndpointConfig.getPassword();
```

Comment:
I believe these can be removed

Reply:
Oh ok I thought EndPointConfig.java is a standard file used by many agents so I didn't change it

Reply:
EndPointConfig.java has been updated to remove the POSTGIS part